.ado/templates/e2e-test-job.yml

```diff
@@ -41,6 +41,13 @@ jobs:
           workingDirectory: $(Build.SourcesDirectory)
         condition: false # Must be manually enabled, since it causes a 5x perf reduction that causes test instability
 
+      # See Issue 6393 for details
+      - powershell: |
+          Invoke-WebRequest https://github.com/microsoft/WinAppDriver/releases/download/v1.1/WindowsApplicationDriver.msi -OutFile $(Agent.TempDirectory)\WinAppDriver.msi
+          Start-Process msiexec -ArgumentList "/quiet","/x","{087BBF93-D9E3-4D27-BDBE-9C702E0066FC}" -Verb runAs -Wait
```

Comment:
We probably don't need the Start-Process and -Verb runAs bits if CI is already running as admin, but these are needed when running locally.